tests/integration_tests/clouds.py

```diff
@@ -143,12 +143,30 @@ def _get_cloud_instance(self):
         )
 
 
+def _mount_source(instance):
+    container_path = '/usr/lib/python3/dist-packages/cloudinit'
```

Comment:
We should remove references to container here, as this works across both containers and VMs. `target_path`, perhaps?